lib/utils.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/unrolled/secure"
 )
 
-const SECURE_MIDDLEWARE_RESTRICTIVE_CONTENT_SECURITY_POLICY = "default-src 'self'"
+const SECURE_MIDDLEWARE_RESTRICTIVE_CONTENT_SECURITY_POLICY = ""
```

Comment:
I'm not a CSP expert, could you elaborate on why do we need to kill this for Postgres?

Reply:
I was having local dev issues for some unknown reason. I'll put this back.